pandas/io/parsers.py

```diff
@@ -1901,6 +1901,11 @@ def __init__(self, src, **kwds):
                     not set(usecols).issubset(self.orig_names)):
                 _validate_usecols_names(usecols, self.orig_names)
 
+            # GH 25623
```

Comment:
can you add a 1-line comment here on what you are validating

Reply:
@jreback does the comment make sense? I am having a weird writer's block...

Reply:
@jreback Added. does the comment make sense?